pageserver/src/tenant/mgr.rs

```diff
@@ -266,8 +326,9 @@ pub async fn shutdown_all_tenants() {
     // On error, log it but continue with the shutdown for other tenants.
     for tenant in tenants_to_freeze_and_flush {
         let tenant_id = tenant.tenant_id();
-        debug!("shutdown tenant {tenant_id}");
+        debug!("freeze_and_flush tenant {tenant_id}");
 
+        // TODO this could probably run in a JoinSet as well?
```

Comment:
I'll do this next, I don't think it's plausible to wait each one-by-one.

Reply:
[f46121e](https://github.com/neondatabase/neon/pull/4321/commits/f46121e869ed6bb8fa70dea407b29b3223764523)